### DISCUSSION TITLE
Close All Files

### DISCUSSION THREAD

**Discussion by Robbie711** [sumatrapdfreader/sumatrapdf#3865] [General]
I browsed through the docs, but couldn't find a Close All Files feature. Is there one -- or did I miss it? (If there isn't one, add that to my wish list, please.)

**Comment by GitHubRulesOK** [comment]
Hmm technically if you have home screen active you could assign a key to close ALL other tabs.
Without home tab, you can select one and close all others, or pick the leftmost (apart from home :-) and close all to right.

**Comment by Robbie711** [comment]
I searched on custom keyboard shortcuts on the Customizing SumatraPDF 3.5.1 page. But actually writing the assignment is beyond my skill set.

**Reply by GitHubRulesOK** [reply]
in advanced settings you can add your choice of keys so if I am on home page and press `w` all tabs close (see example below)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/101178c0-4b03-4707-91ba-ff3b09a7fb7d)

```
Shortcuts [
	[
		Cmd = CmdToggleMangaMode
		Key = M
	]
	[
		Cmd = CmdCloseOtherTabs
		Key = w
	]
]
```

**Comment by Robbie711** [comment]
That's wonderful. Thanks for doing it! 

---steve
